Dockerfile

```diff
@@ -9,9 +9,17 @@ COPY . /src/blockstack
 WORKDIR /src/blockstack
 
 # Install Dependancies
-RUN apt-get update && apt-get install -y python-pip python-dev build-essential apt-utils libssl-dev libffi-dev rng-tools libgmp3-dev sudo software-properties-common sqlite3 lsof
+RUN apt-get update && apt-get install -y python-pip python-dev build-essential apt-utils libssl-dev libffi-dev rng-tools libgmp3-dev sudo software-properties-common sqlite3 lsof curl wget
```

Comment:
Do we need all of these? I remember we were talking one time and you mentioned that we might not need everything here.